cis_v130/section_4.sp

```diff
@@ -20,44 +20,54 @@ locals {
 benchmark "cis_v130_4" {
   title         = "4 Database Services"
   documentation = file("./cis_v130/docs/cis_v130_4.md")
-  tags          = local.cis_v130_4_common_tags
   children = [
     benchmark.cis_v130_4_1,
     benchmark.cis_v130_4_2,
     benchmark.cis_v130_4_3,
     control.cis_v130_4_4,
     control.cis_v130_4_5,
   ]
+
+  tags = merge(local.cis_v130_4_common_tags, {
+    type = "Benchmark"
+  })
 }
 
 benchmark "cis_v130_4_1" {
   title         = "4.1 SQL Server - Auditing"
   documentation = file("./cis_v130/docs/cis_v130_4_1.md")
-  tags          = local.cis_v130_4_1_common_tags
   children = [
     control.cis_v130_4_1_1,
     control.cis_v130_4_1_2,
     control.cis_v130_4_1_3,
   ]
+
+  tags = merge(local.cis_v130_4_1_common_tags, {
+    type    = "Benchmark"
+    service = "Azure/SQLDatabase"
```

Comment:
@cbruno10 would like to take your view here, in `insight` mod we have only `Azure/SQL`, which has reporting for SQL Database, should we retain **`Azure/SQL`** across? or break into different services as updated now in this file?

Reply:
`Azure/SQL` is good in this case, for reference I used the provider name, e.g., `/providers/Microsoft.Sql`.